### ISSUE TITLE
Configure PHPStorm to run Drupal PHPUnit functional tests (linux host)

### ISSUE THREAD

**Issue by mortona42** [lando/php#59]
I want to be able to run Drupal tests within PHPStorm, but have not been able to configure it correctly. Unit and Kernel tests will run, but Functional tests are running into issues like file permissions or accessing urls.

I configured a CLI interpreter to use docker compose and included all configuration files in ~/.lando/compose/SITE, selecting appserver as the service.

When run is selected, it's giving file permission errors:
```
Exception : User warning: mkdir(): Permission Denied
Drupal\Component\PhpStorage\FileStorage->createDirectory()() (Line: 123)
```
When exec is selected:
`the input device is not a TTY`

If I create a CLI interpreter using the docker image, I get:
`Behat\Mink\Exception\ElementNotFoundException : Button with id|name|label|value "Log in" not found.`
This seems like it's not getting the right page at the configured url, http://appserver

Is there another way to get PHPUnit working in PHPStorm?

**Comment by gitressa** [comment]
You could give https://github.com/lando/drupal-contributions/ a try?

The repo has several waiting merge requests, hopefully they can soon get committed. Until then, collecting some info from this issue might get it running for you? https://github.com/lando/drupal-contributions/pull/76

**Comment by mortona42** [comment]
I'm not running the javascript tests yet so not blocked by the chromedriver setup atm. What I'm looking for is instructions to configure PHPstorm to work with lando. My lando setup is similar to drupal contributions.

This is actually a duplicate of #6, so I'm going to close it and move my post over there.

**Comment by gitressa** [comment]
Thanks for clearing that that up, and great that you got it working in the other issue!
